container/src/utilities/helpers.ts

```diff
@@ -7,9 +7,23 @@ export class GenericHelpersClass {
    * @param {any} functionToCheck function to check
    * @returns {boolean}
    */
-  isFunction (functionToCheck: any): boolean {
+  isFunction(functionToCheck: any): boolean {
     return functionToCheck && {}.toString.call(functionToCheck) === '[object Function]';
   }
+
+  /**
+   * Checks whether web component is an attribute or property. In case of attribute, it returns the parsed value.
+   * @param webcomponent 
+   * @returns returns the parsed webcomponent value.
+   */
+  checkWebcomponentValue(webcomponent: any): object | boolean {
+    console.log('test');
+    if (typeof webcomponent === 'string') {
+      return JSON.parse(webcomponent);
+    } else if (typeof webcomponent === 'boolean' || typeof webcomponent === 'object') {
```

Comment:
Is boolean and string treated the same in this case. I thought ex.: `<element test="true"></element> `, true is always string even though looks like boolean. Maybe some context I am missing 🤔

Reply:
You are right, when you do the html structure it is always a string but you can also do it via javascript like:
```const luigiC = document.createElement('luigi-container');
luigiC.webcomponent={selfRegistered: true}
luigiC.viewurl='bla...'
...and so on
```
In that case it can be an object or boolean. That's why we need this check.